scripts/copy-files.js

```diff
@@ -69,9 +69,13 @@ async function createPackageFile() {
   const newPackageData = {
     ...packageDataOther,
     private: false,
-    main: './node/index.js',
-    module: './index.js',
-    typings: './index.d.ts',
+    ...(packageDataOther.main
+      ? {
+          main: './node/index.js',
+          module: './index.js',
+        }
+      : {}),
+    types: './index.d.ts',
```

Comment:
How the these changes relate to the LICENSE of `@material-ui/types`?

Reply:
There are no `main` nor `modules` entrise in the package.json for @material-ui/types

Reply:
> there are no main nor modules entries in the package.json for @material-ui/types

Right, we run this script on types now. Really which we would've just copied the LICENSE file instead of touching build infra.

> typing should be types to match the convention.

These are equivalent as per https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html. What convetion do you mean?

Reply:
> Right, we run this script on types now. Really which we would've just copied the LICENSE file instead of touching build infra.

We could do that too. It was the initial proposal of @lielfr. But I think that he went too far, he added the LICENSE file in all the packages which aren't necessary.

> What convetion do you mean?

The motivation was to consolidate on a single name for the field, avoiding to use two different names.

Reply:
Legal is just annoying :smile: Let me rebase and then we can get this in.

Reply:
I'll simplify the main/module juggling later. There's a better, conventional approach for it.

Reply:
I'm sorry, it probably would have been better if I'd asked first about the desired way to implement it. Just thought that only copying the file to the `types` module isn't worth a PR by itself.

Reply:
> Just thought that only copying the file to the types module isn't worth a PR by itself.

Any change is worth a PR. Don't ever get discouraged by how minor a change might be perceived.